src/test/resources/expected_output/basic/akka/persistence/testkit/EventStorage.java

```diff
@@ -0,0 +1,13 @@
+package akka.persistence.testkit;
+public  interface EventStorage {
+  static public  class JournalPolicies$ implements akka.persistence.testkit.ProcessingPolicy.DefaultPolicies<akka.persistence.testkit.JournalOperation> {
+    /**
+      * Static reference to the singleton instance of this Scala object.
+      */
+    public static final JournalPolicies$ MODULE$ = null;
+    public  akka.persistence.testkit.ProcessingPolicy.DefaultPolicies<akka.persistence.testkit.JournalOperation>.PassAll$ PassAll ()  { throw new RuntimeException(); }
+    public   JournalPolicies$ ()  { throw new RuntimeException(); }
+  }
+  public  void akka$persistence$testkit$EventStorage$_setter_$DefaultPolicy_$eq (akka.persistence.testkit.ProcessingPolicy.DefaultPolicies<akka.persistence.testkit.JournalOperation>.PassAll x$1)  ;
+  public  akka.persistence.testkit.ProcessingPolicy.DefaultPolicies<akka.persistence.testkit.JournalOperation>.PassAll DefaultPolicy ()  ;
```

Comment:
This still looks rather horrible of course, but without this change this would look like:
```
public void akka$persistence$testkit$EventStorage$_setter_$DefaultPolicy_$eq (()akka.persistence.testkit.ProcessingPolicy.DefaultPolicies.PassAll$ x$1);
()akka.persistence.testkit.ProcessingPolicy.DefaultPolicies.PassAll$ DefaultPolicy ()  ;
```

... which confuses the javadoc tool into refusing operation entirely, even with `--ignore-source-errors`